packages/foreman/rubygem-facter/rubygem-facter.spec

```diff
@@ -1,71 +1,45 @@
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name facter
 
-# Disable debuginfo as no native code is packaged (only Reqs)
-%global debug_package %{nil}
-
-Summary: Command and ruby library for gathering system information
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 4.0.51
-Release: 2%{?dist}
-Group: System Environment/Base
-License: ASL 2.0
-URL: https://puppetlabs.com/%{gem_name}
+Name: rubygem-%{gem_name}
+Version: 4.2.10
+Release: 1%{?dist}
+Summary: Facter, a system inventory tool
+License: Apache-2.0
+URL: https://github.com/puppetlabs/facter
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix}rubygem(hocon) >= 1.3
-Requires: %{?scl_prefix}rubygem(hocon) < 2.0
-Requires: %{?scl_prefix}rubygem(thor) >= 1.0.1
-Requires: %{?scl_prefix}rubygem(thor) < 2.0
-
-
-%ifarch %ix86 x86_64 ia64
-Requires: dmidecode
-Requires: pciutils
-Requires: virt-what
-%endif
-Requires: net-tools
-Requires: which
```

Comment:
I believe some of these may still be needed for some facts. Perhaps as recommended dependencies. I'll look at this.